pages/02.Tutorials/10.cinder-multiattach/docs.en.md

```diff
@@ -0,0 +1,161 @@
+---
+title: 'Cinder multi-attach'
+published: true
+date: '27-04-2020 10:11'
+taxonomy:
+    category:
+        - docs
+---
+
+## Cinder multi-attach
+
+### Overview
+
+In this tutorial we will create a cinder volume and attach it to 3 different instances simoultaniously in Read/Write mode using the OCFS2 filesystem.
+
+### Prerequisites
+
+* 3 running instances using Ubuntu sharing one network
+* You know the basics of using the [OpenStack CLI-Tools](../../03.Howtos/02.openstack-cli/docs.en.md).
+* Environment variables are set, like shown in the [API-Access-Tutorial](../02.api-access/docs.en.md).
+
+### Create a cinder multi-attach volume
+
+Multiattach volumes are created by choosing the type `quobyte-multiattach`.
```

Comment:
We should be consistent how to write multi-attach (vs multiattach), so better "Multi-attach volumes" here